src/components/ApiResponse.vue

```diff
@@ -11,11 +11,18 @@
           <div v-if="showCopyMessage" :class="{'copy-message': true, 'error': isError}">{{ message }}</div>
         </transition>
       </div>
+      <br>
+      <div class="copy-container">
+        <v-btn @click="checkDataQuality">Check Data Quality</v-btn>
+        <transition name="fade">
+          <div v-if="showDataMessage" :class="{'copy-message': true, 'error': isError}">{{ message }}</div>
```

Comment:
maybe we shouldn't use the 'copy message' class

Reply:
yes, it is because of the css class we are using. and I am redesigin it by adding persistent data save and fetch here, will update it togeterh, thanks for your pointing out it.